### PR TITLE
dash/datagrid-docs

### PR DESCRIPTION
**Pull request by sebastianbochan**
- [x] - article + schema
- [x] - add API references

### CODE REVIEW

File: docs/datagrid/understanding-dataGrid.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Understanding DataGrid
+
```
Comment (karolkolodziej) [inline docs/datagrid/understanding-dataGrid.md:2]: Remove this line:
image

File: docs/datagrid/understanding-dataGrid.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            cellFormatter: function () {
+                return '<a href="' + this.value + '">URL</a>';
+            }
+        }
+    }
+```
+
+For more information on cell options see the [API reference]().
+
+Editor
```
Comment (karolkolodziej) [inline docs/datagrid/understanding-dataGrid.md:78]: Do we want to call it `editor`?

Comment (sebastianbochan) [reply docs/datagrid/understanding-dataGrid.md:78]: I was hesitating, my first thought was to call it `Cell Editor`, but it sounds a bit strange as well.

I'm open to suggestions.

**Review by vazonik — COMMENTED** [review]
image
1. I would graphically distinguish the head and the rows group.
2. I would change `Resizer` to `Columns Resizer` since we will probably introduce Rows Resizer in the future.

File: docs/datagrid/understanding-dataGrid.md (hunk 2 above)
Comment (vazonik) [reply docs/datagrid/understanding-dataGrid.md:78]: Cell Value Editor? Value Editor?

Comment (sebastianbochan) [reply docs/datagrid/understanding-dataGrid.md:78]: Value editor sounds good

**Review by ddragula — APPROVED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Review by ddragula — APPROVED** [review]

**Pull request merged**